pom.xml

```diff
@@ -50,8 +50,19 @@
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
```

Comment:
Did we add the swagger dependency?

Reply:
Yes, I have integrated swagger now to the project